docs/specification/current.md

```diff
@@ -365,7 +365,7 @@ goal is to know, at call time:
 The CDS Hooks request is augmented to include two prefetch values, where the dictionary
 keys match the request keys (`patient` and `hemoglobin-a1c` in this case).
 
-Note that the missing `user` key indicates that either the CDS Client has decided not to satisfy this particular prefetch template or it was not able to retrieve this prefetched data. The CDS Service is responsible for retrieving this Practitioner data from the FHIR server (if required).
+Note that the missing `userId` key indicates that either the CDS Client has decided not to satisfy this particular prefetch template or it was not able to retrieve this prefetched data. The CDS Service is responsible for retrieving this PractitionerRole data from the FHIR server (if required).
```

Comment:
@dennispatterson and others, T-System is not planning to return PractitionerRole for all contexts. Currently the CDS Service we are working with in the field is using STU3, which does not have PractitionerRole, and is using the UserId to correlate practitioners in the EHR and CDS Service using the Practitioner ID. This is why a new context parameter would be desirable in order to keep backwards compatibility.

Reply:
@cfeltner What do you mean by saying STU3 doesn't have PractitionerRole? https://hl7.org/fhir/STU3/practitionerrole.html

@dennispatterson Yes, Epic is planning to send PractitionerRole where possible, since this is a more appropriate resource for decision support - it describes not only who the person is but in what capacity are they involved. We will use practitioner if an appropriate practitioner role cannot be found. That said, I personally don't feel strongly about examples saying one way or other - it would be nice to use PractitionerRole since like i said it captures both who and what capacity which is better for decision support than just the who. But if you feel strongly that it should be left as Practitioner in the examples, I'm fine with that..

Reply:
@mattvarghese Yes, you are correct. My mistake. PractitionerRole is in STU3. It is just not used in our currently deployed CDS solution.